client/my-sites/email/titan-new-mailbox/index.jsx

```diff
@@ -127,6 +138,26 @@ const TitanNewMailbox = ( {
 						/>
 					</FormLabel>
 					{ hasPasswordError && <FormInputValidation text={ passwordError } isError /> }
+					{ ( showAlternateEmail || hiddenFieldNames.includes( TITAN_PASSWORD_RESET_FIELD ) ) &&
+						! showAlternateEmail && (
```

Comment:
That condition looks strange (we could write it as `(a || b) && !a` which then can be formulated as `a && !a || b && !a` where the first part will always be `false`). Isn't the following enough?

```suggestion
					{ ( hiddenFieldNames.includes( TITAN_PASSWORD_RESET_FIELD ) ) && ! showAlternateEmail ) && (
```